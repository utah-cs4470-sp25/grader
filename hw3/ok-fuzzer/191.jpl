write image E to ""
time show [ M , myfloat]
let a = 180
write image false to "@n"
write image false to "U"
show false
read image "t" to m
