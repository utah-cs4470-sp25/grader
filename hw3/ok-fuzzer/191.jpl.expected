(WriteCmd (VarExpr E) "")
(TimeCmd (ShowCmd (ArrayLiteralExpr (VarExpr M) (VarExpr myfloat))))
(LetCmd (VarLValue a) (IntExpr 180))
(WriteCmd (FalseExpr) "@n")
(WriteCmd (FalseExpr) "U")
(ShowCmd (FalseExpr))
(ReadCmd "t" (VarLValue m))
